modules/end-user-guide/partials/proc_configuring-a-workspace-to-use-a-custom-image-with-an-ide-based-on-the-intellij-platform.adoc

```diff
@@ -0,0 +1,22 @@
+[id="configuring-a-workspace-to-use-a-custom-image-with-an-ide-based-on-the-intellij-platform_{context}"]
+= Configuring a workspace to use a custom image with an IDE based on the IntelliJ Platform
+
+This section describes how to configure a workspace to use an IDE based on the IntelliJ Platform.
+
+.Procedure
+
+. Add the following component to the workspace devfile:
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: "__<URL_to_meta.yaml>__" <1>
+----
++
+<1> __<URL_to_meta.yaml>__: HTTPS resource defining the IDE metadata, see xref:building-images-for-ides-based-on-the-intellij-platform.adoc[].
```

Comment:
```suggestion
<1> `__<URL_to_meta.yaml>__`: HTTPS resource defining the IDE metadata, see xref:building-images-for-ides-based-on-the-intellij-platform.adoc[].
```

Reply:
URLs are not be in monospace (backticks). Monospace is for files and directories, not addresses -- even if the address points to a file.